### PR TITLE
Menu mobile PR

### PR DESCRIPTION
**Pull request by til2to**
## In this Milestone, the following features were included:
- Javascript linters were added
- When the user clicks (or taps) the hamburger button, the mobile menu appears.
- When the user clicks (or taps) the close (X) button, the mobile menu disappears.
- When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears.
- When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+hamburger.addEventListener('click', () => {
+  if (menuList.style.display !== 'block') {
+    menuList.style.display = 'block';
+    logoName.style.display = 'none';
+    hamburger.style.display = 'none';
+  }
+});
+
+closeHamburger.addEventListener('click', () => {
```
Comment (nedjwab) [inline index.js:15]: [ ] When the user clicks (or taps) any option of the mobile menu, the mobile menu should disappear. your menu is still displayed. kindly fix that.🤓

File: index.html [hunk 2]
```diff
@@ -12,342 +12,347 @@
...
-    <header>
-      <a href="#"><h2>Owusu Dennis</h2></a>
-      <div class="menuImg"><img src="./images/menu.png" class="menu" alt="Menu"></div>
+    <header class="main-header">
+      <a class="logo-name" href="#"><h2 class="owusu">Owusu Dennis</h2></a>
 
       <nav>
-        <ul class="menu_items" id="menuList">
+        <button class="menuImg"><img src="./images/menu.png" alt="Menu"></button>
+        <ul class="menu_items" id="menu_items">
```
Comment (nedjwab) [inline index.html:20]: [ ] the close button is missing from your hamburger button. kindly add it to match the Figma design.
    -When the user clicks (or taps) the close (X) button, the mobile menu disappears.   
Your design :
hamte3oi

Figma design : 
hamfigma

**Review by nedjwab — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi @til2to  

Nice job, you are almost there.



## (Highlights) Good Points: :
- Good implementation of some of the project requirements  ✔
- GitHub flow was correctly used ✔.

## (Changes Required) Aspects to improve: ♻️
You did great implementing some of the project requirements, but you still have some issues: 
check the comments below 
 👇👇👇

 
## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html (hunk 2 above)
Comment (til2to) [reply index.html:20]: Alright

File: index.js (hunk 1 above)
Comment (prolajumokeoni) [reply index.js:15]: - [ ] Just as the previous reviewer requested, please when the user clicks (or taps) any option of the mobile menu, the mobile menu should disappear, please fix it see the video below ⬇️ 


https://user-images.githubusercontent.com/69638013/208862795-23545b9f-4305-4bb7-b406-0439158682bb.mp4

**Review by prolajumokeoni — CHANGES_REQUESTED** [review]
Hi @til2to 
Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used 
✔️  Good PR title and Description

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by roshan-bajgain — APPROVED** [review]
Hi @til2to, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️
- PR has a good title and summary. ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**